lib/src/preset/middleware/gas_price.dart

```diff
@@ -46,6 +41,11 @@ UserOperationMiddlewareFn getGasPrice(
 ) {
   return (ctx) async {
     Object? eip1559Error;
+    if (ctx.op.maxFeePerGas > BigInt.zero &&
```

Comment:
looks good.
how the SDK will set those values?

Reply:
You can see the last PR in our SDK. Let me know what do you think